package/win32/make-package.bat

```diff
@@ -291,51 +287,3 @@ echo     set RSTUDIO_VERSION_SUFFIX=-daily+321
 echo     make-package clean electron
 echo.
 exit /b 0
-
-REM For a full package build the package.json file gets modified with the
```

Comment:
For reference, I moved most of this logic into CMake, and now have the Electron build happening in a separate build directory. Let me know if this seems reasonable (we were already doing this on macOS, for what it's worth)